src/index.test.ts

```diff
@@ -0,0 +1,23 @@
+import { gql } from "./index";
+
+it("returns the expected GraphQL query string", () => {
+  const id = 12345;
+
+  const query = gql`
+    query {
+      product(id: "${id}") {
+        displayName
+      }
+    }
+`;
+
+  const expectedQuery = `
+    query {
+      product(id: "${id}") {
+        displayName
+      }
+    }
+`;
```

Comment:
Just for tests, etc. one handy tip I found to make life easier is install Prettier as a devDep, then have a `format()` function for tests that formats input & output with Prettier. Makes tests more stable as they’re no longer subject to whitespace.

Reply:
I'm not sure I understand what problem this solves?

Reply:
```js  
const query = gql`
    query {
      product(id: "${id}") {
        displayName
      }
    }
`;

const expectedQuery = `
    query {
      product(id: "${id}") {
        displayName
      }
   }
`;
```

Say one person added this as a test case. I think this test would throw, because the whitespace is off by one. Most editors won’t auto-format a templated string. Prettier just makes test-writing easier IMO by testing the values, without making devs manually format whitespace.

That is, if I’m reading your test right I _think_ whitespace matters! But LMK if I’m wrong.